index.html

```diff
@@ -67,227 +78,257 @@ <h2 class="my-position">I’am a Software Developer</h2>
+                    </li>
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">bootstrap</button>
+                    </li>
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">Ruby</button>
+                    </li>
+                </ul>
+                <button type="submit" class="button2 button1-text">See Project</button>
             </div>
-            <div class="language-names">
-                <ul class="language-list">
-                    <li class="language-list-content">
-                        Codekit
+            <div class="page2">
+                <h2 class="heading2 text-color h2-margin">Profesional Art Printing Data</h2>
+                <p class="project-discription text-color">A daily selection of privately personalized reads; no accounts
+                    or
+                    sign-ups required. has been the industry's standard</p>
+                <ul class="web-icons">
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">html</button>
                     </li>
-                    <li class="language-list-content">
-                        Github
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">bootstrap</button>
                     </li>
-                    <li class="language-list-content">
-                        Codepen
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">Ruby</button>
                     </li>
-                    <li class="language-list-content">
-                        Gitlab
+                </ul>
+                <button type="submit" class="button2 button1-text">See Project</button>
+            </div>
+            <div class="page2">
+                <h2 class="heading2 text-color h2-margin">Profesional Art Printing Data</h2>
+                <p class="project-discription text-color">A daily selection of privately personalized reads; no accounts
+                    or
+                    sign-ups required. has been the industry's standard</p>
+                <ul class="web-icons">
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">html</button>
                     </li>
-                    <li class="language-list-content">
-                        Terminal
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">bootstrap</button>
+                    </li>
+                    <li>
+                        <button type="submit" class="web-icon-page2 icon-web">Ruby</button>
                     </li>
                 </ul>
+                <button type="submit" class="button2 button1-text">See Project</button>
+            </div>
+        </div>
+    </section>
```

Comment:
- [ ] Again good job with your cards' project design. So, I noticed from the Figma template that when the project is **hovered on with the mouse the description and dark opacity disappears** and a `See Project` button should appear at the bottom of the project's image. I suggest you fix the issues to make your design stick to the template as much as possible. I know you can do it :muscle: :muscle: But great job.
### Your design
![fig](https://user-images.githubusercontent.com/83042742/171962780-5bba84f5-39b2-4669-9fb3-4fe62428f9af.png)
### Figma design
![Fig](https://user-images.githubusercontent.com/83042742/171962918-133bc3a6-339b-44f0-b9e7-badd4edeeb09.png)